src/bin/jan.rs

```diff
@@ -9,15 +9,32 @@ struct Args {
     /// Whether to decompress instead of compress.
     #[arg(short, long)]
     decompress: bool,
+
+    /// Flavor of the algorithm to use. Defaults to 8bit which is many times faster but slightly worse at compressing.
+    #[clap(value_enum)]
+    #[arg(short, long, default_value = "bit8")]
```

Comment:
The description and the code disagree over the default value: `"8bit" != "bit8"` :)

Reply:
Whoops